### DISCUSSION TITLE
Problem with cheats not working in game

### DISCUSSION THREAD

**Discussion by WarDaddy550** [winstxnhdw/lc-hax#204] [General]
hi i have done everything correctly as ive have installed and opened everything the same way my friend did but when i open launch.bat anti kick does not show in the top right of my screen and none of the commands work in game. thank you for taking time to read this

**Comment by winstxnhdw** [comment]
Send a screenshot of your terminal?

**Reply by WarDaddy550** [reply]
![Untitled](https://github.com/winstxnhdw/lc-hax/assets/156112636/d6738d06-2f1b-44a0-91ad-cf88f810f31f)

**Reply by WarDaddy550** [reply]
is this ok?

**Reply by WarDaddy550** [reply]
![Untitled1](https://github.com/winstxnhdw/lc-hax/assets/156112636/9001525f-e5e6-4c52-8112-2a174aa0dac9)

**Reply by WarDaddy550** [reply]
i have just ran the requirement.bat again then tried to open launch.bat now its saying this

**Reply by winstxnhdw** [reply]
did you recursively clone the repository?

**Reply by WarDaddy550** [reply]
how do i do that

**Reply by winstxnhdw** [reply]
did you read the README?

**Reply by WarDaddy550** [reply]
no sorry i just did everything my friend told me to do as the mod is working for him it was also working for me yesterday aswell then i shut the game then started it again and the cheat wasnt working

**Reply by winstxnhdw** [reply]
then it's probably your anti-virus deleting SharpMonoInjector

**Reply by WarDaddy550** [reply]
ok so should i turn off anti virus then try launch the cheat again

**Reply by winstxnhdw** [reply]
That'll work but I don't recommend doing that. You should instead run the following PowerShell command within the folder to add an exception.

https://github.com/winstxnhdw/lc-hax#whitelisting-sharpmonoinjectorcore

**Reply by WarDaddy550** [reply]
sorry i dont really know how to do that what is powershell

**Reply by WarDaddy550** [reply]
it doesnt matter ive sorted it the cheats are now working for me thank you for your help and keep up the good work i love it

**Comment by WarDaddy550** [comment]
![Untitled](https://github.com/winstxnhdw/lc-hax/assets/156112636/5fa84beb-2306-435f-bcca-cec5195444b8)
